### DISCUSSION TITLE
黑客松 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#475] [General]
# 黑客松 - Miao Yu | 于淼

https://yufree.cn/cn/2024/08/24/hackathon/

**Comment by iotanano** [comment]
线下同步交流碰撞，组织方也能提供更有帮助的支持资源，确实会比线上氛围好。医学领域国内有基于数据库的 datathon，如 301 医院跟 MIT 每年会联合举办，宣传数据库的同时提供很多培训。

类似 datathon 还有 docathons，文档马拉松，也挺好玩，之前在一位老哥的[博客](https://ericmjl.github.io/blog/2024/6/30/two-years-of-docathons-insights-and-lessons-learned/)上看到过介绍，对线下活动策划蛮有启发。

**Reply by yufree** [reply]
说到组织方，确实很重要。这次活动系里有行政助理全程安排，会议中的交流记录也是她做的，会后还给我们这些参与者发了总结，对我这种不记笔记的比较友好，当然她工资的一部分也是我们项目出的，一般一个助理要跟进多个项目的管理。虽然我觉得AI也可以做的一部分功能替代，但做记录本身也可以是学术训练的一部分，可以找研究生来做，线下会议组织这些单靠我们这伙书呆子是搞不明白的。
